gini/gini-switch-sdk-ios
# Delegating errors

# Introduction

I decided to remove the `didFailWithError` delegate method - we will only use `didReceiveError`. Also, the `didReceiveError ` wasn't being called.

# How to test

Use invalid credentials to run the sample app (Make sure there's no valid token cached on the device). Verify that an alert view is shown when you start the SDK. Look at the console logs and verify that there's one saying "Switch SDK did receive an error:".

# Merge info

Automatic.
